discretize/utils/meshutils.py

```diff
@@ -173,17 +173,16 @@ def closestPoints(mesh, pts, gridLoc='CC'):
 
 
 def ExtractCoreMesh(xyzlim, mesh, meshType='tensor'):
-    """Extracts Core Mesh from Global mesh
+    """
+    Extracts Core Mesh from Global mesh
 
     :param numpy.ndarray xyzlim: 2D array [ndim x 2]
     :param BaseMesh mesh: The mesh
 
     This function ouputs::
 
         - actind: corresponding boolean index from global to core
-        - meshcore: core mesh
-
-    Warning: 1D and 2D has not been tested
+        - meshcore: core sdiscretize mesh
     """
     import discretize
```

Comment:
@thast: adding a test would be awesome! I will hold off on merging this until we add one. Would you be game to do that??

@rowanc1: agreed, later on, we should think about making this a method on the mesh, but for now, I think the scope of this pr is to get the util cleaned up so that we are at least back to baseline functionality.